templates/simplesaml-configuration/authsources.php

```diff
@@ -107,7 +107,7 @@
         'capture_peer_cert' => true,
         'allow_self_signed' => false,
         'verify_depth' => 5,
-        'peer_name' => 'as.ite.logon.realme.govt.nz',
```

Comment:
Hey @DarrenInwood, I'm happy with the other changes, but are you able to remove this ITE change?

ITE changes also require a different certificate to be added (with a different filename) and for the `saml20-idp-remote.php` file to be updated with the new service and artifact resolution URLs, so we may as well handle it all as one separate pull request if that makes sense?